docker-compose.yml

```diff
@@ -181,6 +181,13 @@ services:
     ports: []
     depends_on: []
 
+  api-webpack:
+    <<: *API_WEBPACK
+    entrypoint: .docker/deps_ok_then ./node_modules/.bin/webpack
+    command: ''
+    ports: []
+    depends_on: []
```

Comment:
Since we're not inheriting values for `ports` or `depends_on`, do we need to set them to empty?

Reply:
Oh good point, I had just copy-pasted from the `webpack` container config, I'll remove those.